recipes/example/meta.yaml

```diff
@@ -45,6 +45,7 @@ about:
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
   license: MIT
+  license_family: MIT
```

Comment:
No worries. `license_family` is optional, but encouraged. However, there has been a strong push a couple of times to make it required. So at least showing it in the `example` with an optional note is some level of compromise.